turbopack/crates/turbopack-tests/tests/execution/turbopack/resolving/dynamic-unrelated/input/index.js

```diff
@@ -0,0 +1,7 @@
+function requireSub(s) {
+  return require(`./sub/${s}`);
```

Comment:
I feel like completely ignoring these parse errors here will lead to cases where people do not get errors for accidental typos. I think we should at least emit a warning here.